Chat/ClientApp/src/App.tsx

```diff
@@ -32,7 +32,7 @@ export default (): JSX.Element => {
     } else if (page === 'chat') {
       return (
         <ChatScreen
-          removedFromThreadHandler={() => setPage('removedFromThread')}
+          removedFromThreadHandler={() => setPage('chat')}
```

Comment:
why are we going to chat instead of the removedFromThread page

Reply:
Basically this function is called whenever a user is kicked out of a chat thread. In our case we intentionally remove the user when he leaves the room. 
It is not ideal, i believe other alternatives will be alot more complicated to implement.